db/data_listeners.cc

```diff
@@ -59,11 +59,11 @@ void data_listeners::on_write(const schema_ptr& s, const frozen_mutation& m) {
 
 toppartitions_item_key::operator sstring() const {
     std::ostringstream oss;
-    oss << key.key().with_schema(*schema);
+    oss << "(" << schema->ks_name() << ":" << schema->cf_name() << ") " << key.key().with_schema(*schema);
```

Comment:
wouldn't it change current behaviour? it would now add ks/table for users who call nodetool toppartitions for a specific table

Reply:
That was the goal, as we can now have partitions from different tables that have the same names - it's good to have a way to differentiate between them.

Reply:
this is true for someone that uses the new functionality, what about backwards compatibility for users who use the old functionality?

Reply:
changed it to include backwards compatibility